.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: node_js
-node_js: 9
-
+node_js: 10
 install:
-  - cd extension; npm install; cd ..
-
+- npm install -g standard
+- cd extension; npm install; cd ..
 script:
-  - cd extension; npm audit; cd ..
-  - cd extension; npm run coverage; cd ..
+- cd extension; npm audit; cd ..
+- cd extension; standard; cd ..
```

Comment:
There are messages standard is producing but this is not failing travis build and it should.